services/frontend-service/src/ui/components/navigation/navListItem.tsx

```diff
@@ -54,7 +54,7 @@ export const NavListItem = (props: { className?: string; to: string; icon?: JSX.
                     className
                 )}
                 ref={control}
-                to={to}
+                to={to + (!!search ? search : '')}
```

Comment:
```suggestion
                to={`{${to}${(!!search || '')}`}
```
Please use `` instead of +

Reply:
This causes for some weird errors to appear like clicking on the locks page and displaying the path /v2/home. What I did was `to={`${to}${queryParams ? queryParams : ''}`}`